packages/bpk-component-nudger/README.md

```diff
@@ -59,6 +59,7 @@ class App extends Component {
 | onChange              | func                          | true     | -             |
 | value                 | number                        | true     | -             |
 | className             | string                        | false    | null          |
+| buttonType            | string                        | false    | secondary     |
```

Comment:
Could we change `string` to `oneOf('secondary', 'outline')`?

Reply:
should be updated